.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+test
```

Comment:
- [ ] [OPTIONAL] The file name is ```test.md``` please consider using the proper naming so git will not push the file to Github, this is required